### ISSUE TITLE
A possible bug in attention mask

### ISSUE THREAD

**Issue by Summit3Zz** [QwenLM/Qwen3-VL#1506]
in Qwen2_5_VLForConditionalGeneration's forward attention mask type and shape: <class 'torch.Tensor'> torch.Size([2]) 

and in qwen-vl-finetune/qwenvl/data/data_qwen.py:
data_dict["attention_mask"] = [data_dict["input_ids"][0].size(0)]   is this right? why is it like this? The data_dict["attention_mask"]  is a list with only 1 element.

**Comment by JJJYmmm** [comment]
that's used for the sequence packing, as shown below (actually, it's `seqlen` rather than `mask` 🤥

https://github.com/QwenLM/Qwen3-VL/blob/dba33b5b9c8cb80e7b9c0aac854847e082e15f24/qwen-vl-finetune/qwenvl/data/data_qwen.py#L572-L580

the usual mask would be caculated correctly here when packing is not abled.

https://github.com/QwenLM/Qwen3-VL/blob/dba33b5b9c8cb80e7b9c0aac854847e082e15f24/qwen-vl-finetune/qwenvl/data/data_qwen.py#L517

**Comment by Summit3Zz** [comment]
thank u, jie
